manoseimas/mps_v2/models.py

```diff
@@ -74,16 +75,16 @@ def __unicode__(self):
 
 class Group(CrawledItem):
     GROUP_TYPES = (
-        ('group', ('Group')),
-        ('committee', ('Committee')),
-        ('commission', ('Commission')),
-        ('fraction', ('Fraction')),
-        ('parliament', ('Parliament')),
+        ('group', _('Group')),
+        ('committee', _('Committee')),
+        ('commission', _('Commission')),
+        ('fraction', _('Fraction')),
+        ('parliament', _('Parliament')),
```

Comment:
It is good practice to not hardcode choice keys and instead move them to class level constants, like:

```python
GROUP = 'group'
GROUP_TYPES = (
    (GROUP, _("Group")),
)
```

This is also in [Django coding style docs](https://docs.djangoproject.com/en/1.8/internals/contributing/writing-code/coding-style/#model-style).

Also, numeric keys are faster and sortable.